### ISSUE TITLE
Missing MQTT "doorbell" event

### ISSUE THREAD

**Issue by rsulzenbacher** [hjdhjd/homebridge-unifi-access#6]
### Homebridge UniFi Access Version

1.0.0

### Homebridge Platform and OS

Ubuntu

### Homebridge Version

1.7.0

### Node Version

20.11

### UniFi OS Version

3.2.12

### UniFi Access Controller Version

1.23.12

### Describe the problem

First of all, thanks for the great plugin! Works like a charm!
I only noticed that the "doorbell" MQTT event was missing.

Couldn't find a place in the code where this event is triggered, so I added it myself.
Added 
`this.controller.mqtt?.publish(this.accessory, "doorbell", "true");`
on line 390 of access-hub.ts and now it's working. 

### Homebridge HBUA JSON configuration

```json
{}
```

### Relevant log output

```shell
...
```

### Acknowledgment that you are only running UniFi OS and UniFi Access releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Appreciate the feedback. To clarify, this is a feature request, not a bug.😄 

Doorbell ring events will be in a future release. The current list of supported MQTT topics is in the documentation and doesn’t include doorbells.

**Comment by rsulzenbacher** [comment]
Ok... but "doorbell" is mentioned in:
https://github.com/hjdhjd/homebridge-unifi-access/blob/main/docs/MQTT.md#mqtt-support

Either way, thanks for adding it!!!

**Comment by rsulzenbacher** [comment]
> Ok... but "doorbell" is mentioned in: https://github.com/hjdhjd/homebridge-unifi-access/blob/main/docs/MQTT.md#mqtt-support
> 
> Either way, thanks for adding it!!!

This commit removed the feature:
https://github.com/hjdhjd/homebridge-unifi-access/commit/3efdb7f259810188e32225dd96d4649ed23a0150#diff-c6d18d6d43bee3b6af3c4ea45902308572d489c34debccca0708ad21d5778ca4L281

**Comment by hjdhjd** [comment]
Appreciate the callout on the documentation oversight on my part - as I stated, the glossary is there and that is the reference documentation for this plugin. Serves me right for copying over my documentation from HBUP. Either way…this feature’s already in my private tree and will be in an upcoming release. Thanks.

And no…that commit didn’t remove the feature. It was never there to begin with. The commit you’re referencing is removing an **already commented out code fragment** that I pulled over from HBUP. 😄 Note line 223 as the beginning of the commented out section.
